### PR TITLE
[Feature/add-calendar-component] Add Calendar to Dashboard

### PR DESCRIPTION
**Pull request by AlanAnguiano**
## Description

This Pull request adds the Calendar component to Dashboard component as well as some modifications in the sidebar in order to allow scrolling the content but not the sidebar

## Evidence

image

## Link Issue (if applicable)
  -

### CODE REVIEW

File: src/components/Dashboard/Calendar.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
+.react-calendar {
+  max-width: 100%;
+  background: white;
+  border: 1px solid #a0a096;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 1.125em;
+  margin-top: 30px;
```
Comment (AlejandroCuriel) [inline src/components/Dashboard/Calendar.css:7]: Sort this alphabetically

File: src/components/Dashboard/Calendar.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.react-calendar button {
+  margin: 0;
+  border: 0;
+  outline: none;
+  display: flex;
```
Comment (AlejandroCuriel) [inline src/components/Dashboard/Calendar.css:37]: Sort this alphabetically

File: src/components/Dashboard/Calendar.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  margin: 0.5em;
+}
+
+.react-calendar,
+.react-calendar *,
+.react-calendar *:before,
+.react-calendar *:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
```
Comment (AlejandroCuriel) [inline src/components/Dashboard/Calendar.css:30]: Sort this alphabetically

File: src/components/Dashboard/Calendar.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+.react-calendar button:enabled:hover {
+  cursor: pointer;
+}
+
+.react-calendar__navigation {
+  display: flex;
+  height: 44px;
+  margin-bottom: 1em;
```
Comment (AlejandroCuriel) [inline src/components/Dashboard/Calendar.css:51]: Sort this alphabetically

File: src/components/Dashboard/Calendar.css [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+.react-calendar__navigation button {
+  min-width: 44px;
+  background: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (AlejandroCuriel) [inline src/components/Dashboard/Calendar.css:59]: Sort this and the rest alphabetically please

**Review by AlejandroCuriel — CHANGES_REQUESTED** [review]
Please check the styles of your Calendar.css file and order the classes alphabetically ascending

**Review by Usiel-03A7 — APPROVED** [review]

**Pull request closed without merging**